packages/flutter/lib/src/rendering/node.dart

```diff
@@ -79,7 +82,7 @@ class AbstractNode {
   /// Typically called only from the parent's detach(), and to mark the root of
   /// a tree detached.
   void detach() {
-    _attached = false;
+    _owner = null;
```

Comment:
We can, although the inverse assertion (that _owner == null in attach) seems to fail.